imports/plugins/included/payments-paystack/client/settings/components/paystackSettingsForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import _ from "lodash";
+import { TextField, Translation, Checkbox } from "/imports/plugins/core/ui/client/components";
+
+class PaystackSettingsForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      settings: {
+        publicKey: props.settings.publicKey,
+        secretKey: props.settings.secretKey,
+        support: props.settings.support
+      },
+      checkbox: {
+        "Authorize": _.includes(props.settings.support, "Authorize"),
+        "De-authorize": _.includes(props.settings.support, "De-authorize"),
+        "Capture": _.includes(props.settings.support, "Capture"),
+        "Refund": _.includes(props.settings.support, "Refund")
+      }
+    };
+  }
+
+  handleStateChange = e => {
+    const { settings } = this.state;
+    settings[e.target.name] = e.target.value;
+    this.setState({ settings });
+  };
+
+  handleCheckBox = (event, isInputChecked, name) => {
+    const { checkbox, settings } = this.state;
+    checkbox[name] = isInputChecked;
+    this.setState({ checkbox });
+    if (!_.includes(settings.support, name) && isInputChecked) {
+      settings.support.push(name);
+      return this.setState({ settings });
+    }
+    const index = settings.support.indexOf(name);
+    settings.support.splice(index, 1);
+    return this.setState({ settings });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    return this.props.onSubmit(this.state.settings);
+  };
+
+  render() {
+    const { settings } = this.props;
+    const setting = this.state.settings;
+
+    return (
+      <div>
+        {!settings.secretKey &&
+          !settings.publicKey && (
+            <div className="alert alert-info">
+              <Translation defaultValue="PaystackCredentials" i18nKey="admin.paymentSettings.paystackCredentials" />
+            </div>
+          )}
+
+        <form onSubmit={this.handleSubmit}>
+          <TextField label="Secret Key" name="secretKey" type="text" onChange={this.handleStateChange}
+            value={setting.secretKey}
+          />
+          <TextField label="Public Key" name="publicKey" type="text" onChange={this.handleStateChange}
+            value={setting.publicKey}
+          />
+
+          <label className="control-label">
+            <Translation defaultValue="Payment provider supported methods" i18nKey="reaction-payments.paymentSettings.supportedMethodsLabel" />
+          </label>
+          <br />
+
+          <div>
+            <Checkbox label="Authorize" onChange={this.handleCheckBox} name="Authorize" checked={this.state.checkbox.Authorize} />
+          </div>
+
+          {/* <div>
```

Comment:
Good job working on this PR. It looks good so far. However, can you take out this commented out section? More so, why do we have commented out code in source control?

Reply:
Wrote some codes which weren't working as supposed to and inorder to fix the 
bug I had to debug by creating different instances of the code and  work through each of them one after the order. I guess I forgot to remove this particular piece